tests/app/test_app.py

```diff
@@ -451,9 +452,9 @@ def test_if_user_unknown_in_party_service_no_email_sent(self, mock_alerter, mock
                                                                        "status": "ACTIVE",
                                                                        "sampleUnitType": "BI"}))
     @patch.object(CaseServiceMock, 'store_case_event')
-    @patch.object(message_logger, 'info')
-    def test_if_user_has_no_first_name_or_last_name_then_unknown_user_passed_to_case_service(self, mock_logger,
-                                                                                             mock_case, mock_party):
+    @patch.object(message_logger, 'warning')
```

Comment:
Why patch the logger when there is a perfectly good assertLogs method in unittest? https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertLogs